src/network/networkd-dhcp-common.c

```diff
@@ -126,6 +126,38 @@ int config_parse_dhcp_use_sip(
         return 0;
 }
 
+int config_parse_dhcp_use_log(
```

Comment:
It seems not necessary to introduce own config parser but please use `config_parse_bool()`.